src/cli/sandbox.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+* (C) 2019
```

Comment:
This should have your name or a company name, whoever owns copyright on this work.